interfaces/__template__/v0/interface.yaml

```diff
@@ -0,0 +1,33 @@
+# `name` of the interface, as it should be referenced by charms.
+# This should be the same as the name of the interface specification directory.
+# When proposing a new interface, opt for underscores over dashed.
+# The name corresponds to the line with the arrow in the charm `metadata.yaml` snippet below:
```

Comment:
It's so annoying to refer to metadata in docs these days :disappointed:. I've found it easiest to just say "metadata" and leave it to the reader to substitute in `charmcraft.yaml` or `metadata.yaml`.

```suggestion
# The name corresponds to the line with the arrow in the charm metadata snippet below:
```